pkgs/servers/web-apps/discourse/plugins/discourse-prometheus/prometheus_exporter_version

```diff
@@ -0,0 +1 @@
+0.5.0
```

Comment:
It is not very obvious why we need this file. Could it be also generated on the fly?

Reply:
It's imported from the plugin's repo by the `update.py` script and read by the `Gemfile`.